include/seqan3/alphabet/gap/gapped.hpp

```diff
@@ -41,4 +41,54 @@ template <typename alphabet_t>
 //!\endcond
 using gapped = union_composition<alphabet_t, gap>;
 
+// ---------------------------------------------------------------------------------------------------------------------
+// ungapped metafunction
+// ---------------------------------------------------------------------------------------------------------------------
+
+//!\brief Transformation trait that exposes the underlying (ungapped) alphabet type [**default**: no type member].
+//!\ingroup gap
+template <typename t>
+struct ungapped {};
```

Comment:
braces on a new line

Reply:
I am a little confused by the name. There is  a gapped alphabet and a ungapped type trait? 
I would rather follow the STL here and call it `remove_gap`.

Reply:
@smehringer This is gone right?

Reply:
yes